doc/classes/AudioEffectBandLimitFilter.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectBandLimitFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a Band Limit Filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect limits the frequences surrounding a range around the [code]Cutoff Hz[/code], and lets only the frequences out of this range pass.
```

Comment:
frequencies?

Reply:
Also, just so that I don't have an extra request, shall I remove the description for the methods too?

Reply:
I don't think there would be any issues leaving it as is. It's up to you.

Still, we need docs team to review this first, to see if the changes comply with guidelines.

Reply:
The method descriptions are not necessary - they'll go away. However, if you are going to do them (for now, while they are visible), please follow the proper style: "Returns" not "Return", etc.

Reply:
I followed the description of [Sprite](https://github.com/godotengine/godot/blob/master/doc/classes/Sprite.xml#L25) which was approved as
>Return the amount of horizontal frames. See [method set_hframes].

Reply:
That wasn't approved, it was ignored because that's a getter that's going away in favor of a member variable.